packages/openneuro-server/src/graphql/utils/file.js

```diff
@@ -29,10 +29,13 @@ export function UpdatedFile(filepath, size) {
   if (size) this.size = size
 }
 
-export const filterRemovedAnnexObjects = datasetId => async files => {
-  const removedAnnexObjectKeys = (
-    await BadAnnexObject.find({ datasetId }).exec()
-  ).map(({ annexKey }) => annexKey)
-  // keep files that havent had their annex objects removed
-  return files.filter(({ key }) => !removedAnnexObjectKeys.includes(key))
-}
+export const filterRemovedAnnexObjects =
+  (datasetId, userInfo) => async files => {
+    const removedAnnexObjectKeys = (
+      await BadAnnexObject.find({ datasetId }).exec()
+    ).map(({ annexKey }) => annexKey)
+    // keep files that havent had their annex objects removed
+    return userInfo.admin
```

Comment:
This crashes the resolver for anonymous users.

Reply:
hmm. this was something @david-nishi helped with. I was not able to see files that were "flagged" by a dataset owner as admin.

Reply:
@david-nishi pushed a fix